package.json

```diff
@@ -47,22 +47,22 @@
     "registry": "https://registry.npmjs.org/"
   },
   "devDependencies": {
```

Comment:
I didn't (yet) update `@release-it/conventional-changelog` and `commitlint` because they may come with breaking changes, `prettier` because it could make the diff harder to read due to formatting changes and `typescript` because it may highlight new errors.

Reply:
smart choice!